app_spec/test/test.js

```diff
@@ -75,6 +75,45 @@ test('create_post', (t) => {
   t.equal(result.Ok, "QmY6MfiuhHnQ1kg7RwNZJNUQhwDxTFL45AAPnpJMNPEoxk")
 })
 
+test('delete_post', (t) => {
+  t.plan(3)
+
+  const content = "Hello Holo world 321"
+  const in_reply_to = null
+  const params = { content, in_reply_to }
+  const createResult = app.call("blog", "main", "create_post", params)
+
+  t.ok(createResult.Ok)
+
+  const deletionParams = { post_address: createResult.Ok }
+  const deletionResult = app.call("blog", "main", "delete_post", deletionParams)
+
+  t.equals(deletionResult.Ok, null)
+
+  const paramsGet = { post_address: createResult.Ok }
+  const result = app.call("blog", "main", "get_post", paramsGet)
+
+  t.equals(result.Ok, null)
+})
+
+test('update_post', (t) => {
+  t.plan(2)
+
+  const content = "Hello Holo world 123"
+  const in_reply_to = null
+  const params = { content, in_reply_to }
+  const createResult = app.call("blog", "main", "create_post", params)
+
+  t.ok(createResult.Ok)
+
+  const updateParams = { post_address: createResult.Ok, new_content: "Hello Holo" }
+  const result = app.call("blog", "main", "update_post", updateParams)
+
+  t.equals(result.Ok, null)
+})
```

Comment:
For this test to prove that update works it also needs one more part, which is to do a get on the address of the initial post, and show that what comes back is the updated post.

Reply:
But doesn't the address change when updating the contents?